src/options/schema.ts

```diff
@@ -62,8 +62,9 @@ export const optionsSchema = {
         'config/schema.json',
         '../schema.json',
         '/app/config/schema.json',
+        null,
       ],
-      type: 'string',
+      type: ['string', 'null'],
```

Comment:
Similar question here: Does this include `undefined`?

Reply:
no. `undefined` is not valid JSON so not covered by `json-schema`

Reply:
interesting, didn't know that actually, TIL 💡